queries/task_queries/Task_1.sql

```diff
@@ -0,0 +1,4 @@
+﻿Select distinct Bank_name from [bank-task].dbo.Banks
```

Comment:
from [bank-task].dbo.Banks should be on next line (formatting rules)

Also want to see unified sql statement style (better to see all in capslock style: SELECT, WHERE, FROM etc.)

![image](https://user-images.githubusercontent.com/67163370/189294542-f3fd3543-d985-49d6-8c79-444f4ec1a088.png)